packages/@expo/cli/e2e/__tests__/install-test.ts

```diff
@@ -271,7 +278,7 @@ describe('expo-router integration', () => {
     let error: unknown = undefined;
     try {
       // Run `--fix` project dependencies with expo@52 and expo-router from source
-      await executeExpoAsync(projectRoot, ['install', '--fix']);
+      await executeExpoAsync(projectRoot, ['install', '--fix'], { verbose: false });
```

Comment:
Adding `verbose: false`, to prevent the logs from `--fix` being printed to console.

We want the command to fail, so the logs are misleading here